docs/manual-EN/1.overview/3.design-and-architecture/2.storage-design.md

```diff
@@ -19,13 +19,13 @@ As shown in Picture One, there are three layers in Storage Service. The bottom l
 
 Above the local storage engine is the consensus layer that implements multi group raft. Each partition corresponding to a Raft group, is for the data sharding. Currently, **Nebula Graph** uses `hash` to shard data. When creating a space, users need to specify the partition number. Once set, partition number cannot be changed. Generally, the partition number must meet the need to scale-out in the future.
 
-Above the consensus layer is the storage interface that defines a set of APIs that are related to graph. These API requirements are translated into a set of kv operations targeting at the corresponding partition. It is this layer that makes the storage service a real graph storage, otherwise it's just a kv storage. **Nebula Graph** doesn't separate kv as an independent service as graph query involves a lot of calculation that uses schema, which is not existed in kv layer. Such architecture makes pushing down more easily.
+Above the consensus layer is the storage interface that defines a set of APIs that are related to graph. These API requests are translated into a set of kv operations to the corresponding partition. It is this layer that makes the storage service a real graph storage, otherwise it's just a kv storage. **Nebula Graph** doesn't use kv-store as an independent service as a graph query involves a lot of calculation that involves schema, which is not existed in the kv layer. Such architecture makes computation operation pushing down more easily.
 
 ## Schema & Partition
 
-**Nebula Graph** stores the vertices, edges and properties. Efficient property filtering is critical for a Graph Database.
+As a graph database, **Nebula Graph** stores the vertices, edges and their properties. How to Efficiently filtering or projecting is critical for a graph exploration.
 
-**Nebula Graph** uses tags to indicate vertices types. One vertexID can relate with multiple tags, and each tag has its own properties. In kv storage, we use `vertexID + TagID` to represent key, and the related properties are placed behind value after encoding. The format is shown in Picture Two:
+**Nebula Graph** uses **tags** to indicate a vertex type. One vertex can have multiple types (and therefore multiple tags), and each tag defines its own properties. In the kv store, we use `vertex_ID + Tag_ID` together as a key, and the corresponding value are the encoded property. The format is shown in Picture Two:
```

Comment:
please here change Pict.1 to Fig.1. And Picture Two to Figure Two. It conflicts with my pr, and I'll close mine.